README.md

```diff
@@ -34,3 +39,7 @@ If you use this software in a scientific publication, please consider citing one
 
 - Cole Brokamp, Chris Wolfe, Todd Lingren, John Harley, Patrick Ryan. Decentralized and Reproducible Geocoding and Characterization of Community and Environmental Exposures for Multi-Site Studies. *Journal of American Medical Informatics Association*. 25(3). 309-314. 2018. [*Download*](https://colebrokamp-website.s3.amazonaws.com/publications/Brokamp_JAMIA_2017.pdf).
 - Cole Brokamp. DeGAUSS: Decentralized Geomarker Assessment for Multi-Site Studies. *Journal of Open Source Software*. 2018. [*Download*](https://colebrokamp-website.s3.amazonaws.com/publications/Brokamp_JOSS_2018.pdf).
+
+### Used and Trusted By: <p float="left" >
```

Comment:
Can you also add images from:

- eMERGE: https://www.genome.gov/sites/default/files/media/images/2019-03/eMERGE%20logo%20%28full_white%29_0.png
- PEPR: https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fcarragroup.org%2FUserFiles%2Fimage%2Flogopepr.png&f=1&nofb=1
- Harvard University: https://external-content.duckduckgo.com/iu/?u=http%3A%2F%2F1000logos.net%2Fwp-content%2Fuploads%2F2017%2F02%2FHarvard-college-emblems.jpg&f=1&nofb=1
- Northwestern: https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fupload.wikimedia.org%2Fwikipedia%2Fcommons%2Fthumb%2F5%2F56%2FNorthwestern_University_seal.svg%2F1200px-Northwestern_University_seal.svg.png&f=1&nofb=1
- Columbia: https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fmedia.imotions.com%2Fimages%2F20190603123856%2FColumbia-University-Logo-300x300.png&f=1&nofb=1
- Weill Cornell Medicine: https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fbrand.weill.cornell.edu%2Fsites%2Fdefault%2Ffiles%2Flogo_files%2FWCM_MB_LOGO_HZSS2L_CLR_RGB.png&f=1&nofb=1
- ODH: https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fodh.ohio.gov%2Fwps%2Fwcm%2Fconnect%2Fgov%2F8863f3a8-49fa-4569-a41d-2d75b743bf05%2FODH%2Blogo.png%3FMOD%3DAJPERES&f=1&nofb=1
- HCJFS: https://www.hcjfs.org/media/JFS_Logo_JFS_Color.jpg
- St. Jude's: https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse2.mm.bing.net%2Fth%3Fid%3DOIP.QbX8sXMo0EElTVH1PU5sAQHaEc%26pid%3DApi&f=1
- mayo clinic: https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse4.mm.bing.net%2Fth%3Fid%3DOIP.8QLK9UwzEEvwbXwC5ugy6gAAAA%26pid%3DApi&f=1

Reply:
Got it! I will get those added in